src/styles/components/_tag.scss

```diff
@@ -8,9 +8,10 @@
 	height: 3.2rem;
 	padding: $spacer-xs $spacer-sm;
 	border-radius: 1.6rem;
-	font-weight: 700;
 	background-color: $teal-66;
 	color: $black;
+
+	// font-weight: 700; // Tags shouldn't default to being bold
```

Comment:
so this line can be removed?

Reply:
It's a warning to avoid future regression, I'd leave it in. Minifying code is the com-/transpiler's problem, not ours.